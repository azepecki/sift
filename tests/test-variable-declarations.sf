def int main() {

	int x = 2;
	print(x);

	str y = "test str";
	print(y);

	bool a = true;
	print(a);

	char b = 'a';
	print(b);

	return 0;
}