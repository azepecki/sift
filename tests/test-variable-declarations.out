2
test str
true
a